src/V3LinkDot.cpp

```diff
@@ -2455,7 +2451,6 @@ class LinkDotResolveVisitor final : public VNVisitor {
                             m_ds.m_unresolvedClass = true;
                         } else {
                             const auto cextp = classp->extendsp();
-                            UASSERT_OBJ(cextp, nodep, "Bad super extends link");
```

Comment:
Did you intend to remove this?  Seems to prevent a segfault if tree is badly constructed.

Reply:
Yes I just noticed `classp->extendsp() == nullptr` was checked few lines above it.